distribution/dashboard/internal/model/param/backup_param.go

```diff
@@ -1,43 +1,48 @@
 package param
 
+type ScheduleBase struct {
+	// Enum: Weekly, Monthly
+	ScheduleType  string         `json:"scheduleType" example:"Weekly"`
+	ScheduleDates []ScheduleDate `json:"scheduleDates"`
+	// Description: HH:MM
+	// Example: 04:00
+	ScheduleTime string `json:"scheduleTime" example:"04:00"`
+}
+
 type BackupPolicyBase struct {
 	// Enum: NFS, OSS
-	DestType    BackupDestType `json:"destType" binding:"required"`
-	ArchivePath string         `json:"archivePath"`
-	BakDataPath string         `json:"bakDataPath"`
+	DestType    BackupDestType `json:"destType" binding:"required" example:"NFS"`
+	ArchivePath string         `json:"archivePath" binding:"required"`
+	BakDataPath string         `json:"bakDataPath" binding:"required"`
 
-	ScheduleType  string         `json:"scheduleType" binding:"required"`
-	ScheduleDates []ScheduleDate `json:"scheduleDates"`
-	// Description: HH:MM
-	ScheduleTime string `json:"scheduleTime,omitempty"`
+	ScheduleBase `json:",inline"`
 
-	JobKeepWindow  string `json:"jobKeepWindow,omitempty"`
-	RecoveryWindow string `json:"recoveryWindow,omitempty"`
-	PieceInterval  string `json:"pieceInterval,omitempty"`
+	JobKeepWindow  int `json:"jobKeepWindow,omitempty" example:"5"`
```

Comment:
if you'd like to use int for days, at least reflect it in parameter name. jobKeepWindow  -> jobKeepWindowDays

Reply:
OK, I've renamed them to `xxxDays` to get more deterministic.